docs/getting_started.md

```diff
@@ -1,34 +1,30 @@
-#Getting Started with Jaeger
+#Getting Started
 
-We're working hard on making all backend components of Jaeger open
-source. Please watch the GitHub repository
-<https://github.com/uber/jaeger> for updates.
+##All in one Docker image 
+This image, designed for quick local testing, launches the Jaeger ui, collector, query, and agent, with an in memory storage component. 
 
-Jaeger client libraries are already open source and for now can be used
-with Zipkin backend. You can see a HOW-TO example in the blog post
-[Tracing HTTP request latency in Go with
-OpenTracing](https://medium.com/@YuriShkuro/tracing-http-request-latency-in-go-with-opentracing-7cc1282a100a).
+The simplest way to start the all in one docker image is to use the pre-built image published to docker hub.
 
-##Running Jaeger in Docker Container
-
-Coming soon...
+```bash
+docker run -d -p5775:5775/udp -p16686:16686 jaegertracing/all-in-one:latest
```

Comment:
@pavolloffay Just came across this [thread](https://forums.docker.com/t/should-docker-run-net-host-work/14215) where somebody else ran into the same problem last year.

You are right about the ports
`6831` accepts the new model encoded with the `compact` protocol 
`6832` accepts the new model encoded with the `binary` protocol 

See [config.go](https://github.com/uber/jaeger/blob/2179415584f51807709b038cd6c8d7c56ff5666d/cmd/agent/app/config.go#L97) for more details